ONSdigital/eq-translations
# Add support for POT template extraction

**Motivation**

We need to start generating a .pot file with enough context for each string to be translated. 

**How to test**

Run the template extractor against a survey to extract a 'pot' file with strings to translate:

`pipenv run python -m app.template_extractor <survey> <outfolder>`

For answer options: 

- Should capture question text prefixed with "Answer for: " as msgctxt
- Answer id's should be captured in the extracted template as code comments (#:)
